features/cache.feature

```diff
@@ -30,6 +44,27 @@ Feature: Manage oEmbed cache.
       Success: Cleared oEmbed cache.
       """
 
+  Scenario: Clear all oEmbed caches
+    # No oEmbed caches yet
+    When I try `wp embed cache clear --all`
+    Then STDERR should be:
+      """
+      Error: No oEmbed caches to clear!
+      """
+
+    # Generate some various oEmbed caches types
+    When I run `wp post generate --post_type=oembed_cache --post_date="2000-01-01" --post_title=$(wp eval 'echo md5( "foo" );') --count=10`
+    And I run `wp post meta add 1 _oembed_$(wp eval 'echo md5( "foo" );') foo`
+    And I run `wp post meta add 1 _oembed_$(wp eval 'echo md5( "bar" );') bar`
+    # Make sure we don't clear non-oEmbed caches
+    And I run `wp post meta add 1 _oembed_foo bar`
+    And I run `wp transient set oembed_$(wp eval 'echo md5( "bar" );') bar $(wp eval 'echo DAY_IN_SECONDS;')`
+    And I run `wp embed cache clear --all`
+    Then STDOUT should be:
+      """
+      Success: Cleared 3 oEmbed caches: 1 post cache, 1 oembed post cache, 1 transient cache.
+      """
```

Comment:
Can we also add some similar-looking meta, and then include verification that it doesn't get unexpectedly deleted?